packages/hamburger-react/src/Hamburger.tsx

```diff
@@ -1,54 +1,36 @@
-import React, { RefObject, useRef, useState } from "react";
-import { useClickOutside } from "@fremtind/jkl-react-hooks";
+import React from "react";
 
 interface Props {
     className?: string;
-    onClick?: (isActive: boolean) => void;
-    onClickOutside?: (isActive: boolean) => void;
-    enableClickOutside?: boolean;
-    initialIsActive?: boolean;
+    onClick?: () => void;
+    active: boolean;
     negative?: boolean;
     description?: string;
-    insideRef?: RefObject<HTMLElement>;
 }
 
 export const Hamburger = ({
```

Comment:
Denne trenger nok forwardRef, for å kunne benytte seg av clickoutside funksjonalitet utenfor.